server/cluster/metadata/topology_manager.go

```diff
@@ -291,6 +307,64 @@ func (m *TopologyManagerImpl) RemoveTable(ctx context.Context, shardID storage.S
 	return nil
 }
 
+func (m *TopologyManagerImpl) GetTableShard(_ context.Context, table storage.Table) (storage.ShardID, bool) {
```

Comment:
If the fetched result is shard id, how about renaming it as:
```suggestion
func (m *TopologyManagerImpl) GetTableShardID(_ context.Context, table storage.Table) (storage.ShardID, bool) {
```